### ISSUE TITLE
The user and elementary usage.

### ISSUE THREAD

**Issue by peasthope** [corbinlc/GNURootDebian#261]
Hi,

Should ordinary use proceed as root or as a user?  After "adduser me ; cd /home/me ; touch afile" ,
"ls -l afile" reports ownership as root 9777. (?)

Thanks,                       ... Peter E.

**Comment by AdamMatthewLTM** [comment]
@peasthope , 

Thank you for your post. In general on all computing systems it is a good idea to work as root as little as possible - however due to certain permissions requirements GNURootDebian may have the user operate as root when necessary. This may be due to Android's access and permissions requirements.

So you may find yourself operating as root for a significant time using the application - this is intended.

There is a new beta released intended to be an update and large-scale fix for GnuRootDebian known as "UserLAnd" and can be found on GitHub at https://github.com/CypherpunkArmory/UserLAnd . This is still very much a beta however, and updates will be done periodically. If you'd like to give it a look please do so!

I hope I was able to assist you, and thank you,
-Adam

**Comment by peasthope** [comment]
> I hope I was able to assist you, and thank you,

Yes, definitely.  My requirement at present is minimal and I only wanted to confirm that I wasn't misunderstanding elementary usage.

Thanks,                          ... Peter E.
